src/content/packages-and-plugins/using-packages.md

```diff
@@ -124,10 +124,10 @@ To add the package, `css_colors`, to an app:
 
 ### Adding a package dependency to an app using `flutter pub add`
 
-To add the package, `css_colors`, to an app:
+To add the package `css_colors` to an app:
 
 1. Issue the command while being inside the project directory
-   * `flutter pub add css_colors`
+   * `flutter pub add css_colors:'^1.0.0'`
```

Comment:
I think we don't need to specify a version constraint when using `flutter pub add`, as it will automatically add the most recent compatible version using caret syntax.